packages/common/src/pipes/number_pipe.ts

```diff
@@ -17,29 +17,53 @@ import {invalidPipeArgumentError} from './invalid_pipe_argument_error';
  * @ngModule CommonModule
  * @description
  *
- * Transforms a number into a string,
- * formatted according to locale rules that determine group sizing and
- * separator, decimal-point character, and other locale-specific
- * configurations.
+ * Formats a value according to digit options and locale rules.
+ * Locale determines group sizing and separator,
+ * decimal-point character, and other locale-specific configurations.
+ *
+ * <h3>digitsInfo</h3>
+ *
+ * The value's decimal representation is specified by the ```digitsInfo```
```

Comment:
Only use single backticks for inline code blocks - unless I am missing something.
(Similarly below)

```suggestion
 * The value's decimal representation is specified by the `digitsInfo`
```

Reply:
Is there a technical reason for this distinction? Or is it just semantic? It looks like there's no difference in the resultant markup..
![image](https://user-images.githubusercontent.com/61225472/107465346-c3d85a80-6b94-11eb-837b-611bbb9a315e.png)